developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,125 @@ You can request additional properties by sending a request body with the :code:`
++-------------------------------+---------------------------------------------------+---------------------------------------------+
+| <oc:comments-unread />        | The number of unread comments.                    |                                             |
++-------------------------------+---------------------------------------------------+---------------------------------------------+
+| <oc:owner-id />               | The user id of the owner of a shared file.        |                                             |
++-------------------------------+---------------------------------------------------+---------------------------------------------+
+| <oc:owner-display-name />     | The display name of the owner of a shared file.   |                                             |
++-------------------------------+---------------------------------------------------+---------------------------------------------+
+| <oc:share-types />            | The list of type share.                           | | ``0`` = User                              |
+|                               |                                                   | | ``1`` = Group                             |
+|                               |                                                   | | ``3`` = Public link                       |
+|                               |                                                   | | ``4`` = Email                             |
+|                               |                                                   | | ``6`` = Federated cloud share             |
+|                               |                                                   | | ``7`` = Circle                            |
+|                               |                                                   | | ``8`` = Guest                             |
+|                               |                                                   | | ``9`` = Remote group                      |
+|                               |                                                   | | ``7`` = Circle                            |
+|                               |                                                   | | ``10`` = Talk conversation                |
+|                               |                                                   | | ``12`` = Deck                             |
+|                               |                                                   | | ``15`` = Science mesh                     |
++-------------------------------+---------------------------------------------------+---------------------------------------------+
+| <ocs:share-permissions />     | The permissions that the user has over the share. | | ``4`` = Create                            |
+|                               |                                                   | | ``1`` = Read                              |
```

Comment:
Sort the permissions by their value, makes it easier to understand how they are used.